### DISCUSSION TITLE
DNS problem

### DISCUSSION THREAD

**Discussion by Adelscott** [qdm12/gluetun#753] [Ideas]
Since this morning I can't reach news.usenetserver.com through gluetun.
When I try "ping news.usenetserver.com" from the container console I get "bad address 'news.usenetserver.com'"
Same test with "ping news.newshosting.com"   from the container console works, I get an IP.

Right after the container starts it seems to work though, but after a few instants it doesn't anymore. Testing this is a bit annoying as I don't know the command to restart openvpvn from the console, I have to restart the container and rush to the portainer UI that access the container console.

What could be the problem ?

**Comment by Adelscott** [comment]
After a few tests, it seems that the problem occurs only when DNS over TLS is enabled

**Reply by qdm12** [reply]
Yeah Dns over tls (dot) takes a few seconds to start, that's probably why you manage to resolve it at container start time.

Maybe increase the verbosity of the DoT server to see why it's not happy (check the wiki for the env variable).

Anyway, implementation is about to be replaced in the coming week with my own DoT/DoH server so that might just fix it (subscribe to #137)

**Reply by Adelscott** [reply]
Thank you, I'll keep an eye on the new feature!
